amis/build_ami.sh

```diff
@@ -45,8 +44,9 @@ if [ "${public}" == "public" ]; then
   export AMI_PERMS="all"
 fi
 
-# NOTE: the AMI is always built in us-east-1 and then copied to the specified regions
+# NOTE: if region is all then the ami is built in us-east-1 and then copied to the specified regions
```

Comment:
This sentence is wrong at the moment.
We are not setting AWS_REGION anywhere. If we want this value to be automatic assigned, we need to make a decision like:
- if ${region} = all or any other region except govcloud then AWS_REGION=us-east-1, meaning that we always want to build on us-east-1 and then copy the ami to other/s region (this is like we are doing right now)
- if ${region} = govcloud then AWS_REGION=govcloud, meaning that ami build for govcloud need to be built in govcloud